provisioning/resources/ui/js/components/Overview.tsx

```diff
@@ -49,13 +48,14 @@ export class Overview extends React.Component<{}, {}> {
         </ul>
         <h3>The software stack installed: </h3>
         <ul>
-          <li>Snowplow Stream Collector 0.11.0</li>
+          <li>Snowplow Stream Collector NSQ 0.13.0</li>
           <li>Snowplow Stream Enrich NSQ 0.16.1</li>
-          <li>Snowplow Elasticsearch Sink 0.10.1</li>
-          <li>Snowplow Iglu Server 0.2.0</li>
-          <li>NSQ 1.0.0</li>
-          <li>Elasticsearch 1.7.5</li>
-          <li>Kibana 4.0.1</li>
+          <li>Snowplow Elasticsearch Loader 0.10.1</li>
+          <li>Snowplow Iglu Server 0.3.0</li>
+          <li>Postgres 9.5</li>
+          <li>NSQ v1.0.0-compat</li>
+          <li>Elasticsearch 5.6.10</li>
+          <li>Kibana 5.6.10</li>
```

Comment:
aren't we better of with a 6.x version? cc @jbeemster

Reply:
Might as well yes.  Does the latest ES Sink support that?

Reply:
http is fine :+1:

Reply:
this wasn't done

Reply:
Done now